stumpy/mstump.py

```diff
@@ -119,17 +119,20 @@ def _multi_mass(Q, T, m, M_T, Σ_T, μ_Q, σ_Q):
         Window size
 
     M_T : numpy.ndarray
-        Sliding mean for `T_A`
+        Sliding mean for `T`
 
     Σ_T : numpy.ndarray
-        Sliding standard deviation for `T_A`
+        Sliding standard deviation for `T`
 
     μ_Q : numpy.ndarray
         Mean value of `Q`
 
     σ_Q : numpy.ndarray
         Standard deviation of `Q`
 
+    T_subseq_isconstant : numpy.ndarray
+        A boolean array that indicates whether a subsequence in `T_A` is constant (True)
```

Comment:
```suggestion
        A boolean array that indicates whether a subsequence in `T` is constant (True)
```

@seanlaw I forgot to fix this. I will take care of it.